app/views/data_view.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+from app.models import Donor, Donation, Item
+from django.http import HttpResponse, JsonResponse, HttpResponseBadRequest
+from django.views import View
+import simplejson as json
+
+# return aggregate value for given time interval
```

Comment:
Function comments go inside the function. Weird I know but just trying to reinforce the standard